solutions/1500-1599/1514-path-with-maximum-probability.md

```diff
@@ -54,45 +60,54 @@ Explanation: There is no path between 0 and 2.
 * `0 <= succProb[i] <= 1`
 * `There is at most one edge between every two nodes.`
 
-## Approach:  
-
-In standard djikstra algorithm, we will find the shortest path from start to end and update the "minimum distance" array accordingly for each "from -> to" pair.  
-In this question, we created a max heap for finding out maximum probability as asked in the problem statement.  
-The code for this approach is below : 
```

Comment:
Deleting doesn't mean resolving the problem. Please add your explanation back with better format.